src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes, useKeyOnly } from '../../utils/propUtils'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ */
+function BreadcrumbSection(props) {
+  const {
+    active, children, className, link, href, onClick, ...rest,
+  } = props
+
+  const handleClick = (e) => {
+    if (onClick) onClick(e)
```

Comment:
You need to click on the component while it does not have have an `onClick` prop.  You can just assert that it does not throw any errors.

Reference the [Accordion test for onClick](https://github.com/TechnologyAdvice/stardust/blob/master/test/specs/modules/Accordion/Accordion-test.js#L162).